src/main/java/com/google/sps/servlets/ImagesServlet.java

```diff
@@ -0,0 +1,35 @@
+package main.java.com.google.sps.servlets;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/images")
+public class ImagesServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String projectId = "spring21-sps-2";
+        String bucketName = "spring21-sps-2";
+        Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
+        Bucket bucket = storage.get(bucketName);
+        Page<Blob> blobs = bucket.list();
+        
+
+
+        response.setContentType("text/html;");
+        for (Blob blob : blobs.iterateAll()){
```

Comment:
This is okay, but just for reference we can also do it in JSON which might be easier. I've done the implementation on my portfolio, and it's not too bad. I could handle it if it need be. Though not a real issue for now.

Reply:
Good call. Feel free to create a task for that in Trello.